src/modules/airspeed_selector/airspeed_selector_params.c

```diff
@@ -194,3 +194,17 @@ PARAM_DEFINE_INT32(ASPD_FS_T2, 100);
  * @unit m/s
  */
 PARAM_DEFINE_FLOAT(ASPD_STALL, 10.0f);
+
+/**
+ * Maximal airspeed allowed for arming.
```

Comment:
```suggestion
 * Maximum airspeed allowed for arming.
```

Reply:
@sfuhrer Actually this seems to be correct, although it sound weird to me.

Reply:
true, fixed.